notebooks/photutils/03_aperture_photometry/03_aperture_photometry.ipynb

```diff
@@ -0,0 +1,970 @@
+    "                      norm=norm_image)\n",
+    "\n",
+    "# Plot the apertures\n",
+    "for aperture in elliptical_annuli:\n",
+    "    aperture.plot(color='red', alpha=0.4, ax=ax1, fill=True)\n",
+    "for aperture in elliptical_apertures:\n",
+    "    aperture.plot(color='white', alpha=0.7, ax=ax1)\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar = plt.colorbar(fitsplot, fraction=0.046, pad=0.04, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "\n",
+    "# Define labels\n",
+    "cbar.set_label(r'Flux Count Rate ({})'.format(xdf_image.unit.to_string('latex')), \n",
+    "               rotation=270, labelpad=30)\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('Elliptical Annuli')\n",
+    "\n",
+    "# Crop to show an inset of the data\n",
+    "ax1.set_xlim(2000, 3000)\n",
+    "ax1.set_ylim(2000, 1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calculating Aperture Corrections\n",
+    "\n",
+    "Now that our apertures have been defined, we can do photometry with them to estimate and account for the background. The aperture correction is calculated by:\n",
+    "- Calculating the count rate within each annulus using `aperture_photometry`\n",
+    "- Dividing each annulus' count rate by each annulus' area to get the mean background value for each annulus\n",
+    "- Taking the mean of those annulus means to get a mean background value for the entire image\n",
+    "- Multiplying the global background mean value times the area of each elliptical photometric aperture, to get the estimated background count rate within each aperture\n",
+    "- Subtracting the estimated background count rate from the photometric count rate for each aperture\n",
+    "\n",
+    "(Just like when we did photometry with the elliptical apertures above, the below step will take almost 5 minutes.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The CCDData mask will be automatically applied\n",
+    "bkg_phot_table = aperture_photometry(xdf_image, elliptical_annuli[0])\n",
+    "idx = 1\n",
+    "for aperture in elliptical_annuli[1:]:\n",
+    "    idx += 1\n",
+    "    phot_row = aperture_photometry(xdf_image, aperture)[0]\n",
+    "    phot_row[0] = idx\n",
+    "    bkg_phot_table.add_row(phot_row)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Display table\n",
+    "bkg_phot_table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Calculate the mean background level (per pixel) in the annuli \n",
+    "bkg_area = [annulus.area() for annulus in elliptical_annuli]\n",
```

Comment:
This line gives me an error `TypeError: 'float' object is not callable`. I will try to dig into why it's doing that if I get time later, but for now that's as far as I can get in the cells.

Reply:
All that needed changing here was `annulus.area()` -> `annulus.area`, where it appears `area` changed from a function to a property.